src/types/FilterType.ts

```diff
@@ -0,0 +1,6 @@
+export type FilterType = 'all' | 'active' | 'completed';
+export const FILTER_TYPES = {
+  ALL: 'all',
+  ACTIVE: 'active',
+  COMPLETED: 'completed',
+};
```

Comment:
Use enum here instead of object 
<img width="324" alt="image" src="https://github.com/user-attachments/assets/81438dcf-6d3a-4c6a-bb0d-cbce942c36a4" />